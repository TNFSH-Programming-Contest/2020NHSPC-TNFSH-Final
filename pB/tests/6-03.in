904
891.421 -380.174
888.757 -386.361
886.05 -392.529
883.3 -398.678
880.508 -404.807
877.673 -410.917
874.796 -417.007
871.876 -423.078
868.915 -429.127
865.911 -435.156
862.866 -441.164
859.779 -447.151
856.65 -453.116
853.48 -459.059
850.269 -464.98
847.017 -470.878
843.723 -476.754
840.389 -482.606
837.015 -488.436
833.6 -494.242
830.144 -500.023
826.649 -505.781
823.114 -511.514
819.539 -517.223
815.924 -522.907
812.27 -528.565
808.576 -534.198
804.844 -539.805
801.073 -545.386
797.263 -550.94
793.414 -556.468
789.527 -561.969
785.602 -567.443
781.64 -572.89
777.639 -578.309
773.601 -583.7
769.525 -589.062
765.412 -594.397
761.262 -599.702
757.076 -604.979
752.853 -610.226
748.593 -615.444
744.298 -620.632
739.966 -625.79
735.599 -630.918
731.196 -636.016
726.758 -641.082
722.284 -646.118
717.776 -651.123
713.233 -656.096
708.656 -661.037
704.044 -665.946
699.399 -670.824
694.719 -675.669
690.006 -680.481
685.26 -685.26
680.481 -690.006
675.669 -694.719
670.824 -699.399
665.946 -704.044
661.037 -708.656
656.096 -713.233
651.123 -717.776
646.118 -722.284
641.082 -726.758
636.016 -731.196
630.918 -735.599
625.79 -739.966
620.632 -744.298
615.444 -748.593
610.226 -752.853
604.979 -757.076
599.702 -761.262
594.397 -765.412
589.062 -769.525
583.7 -773.601
578.309 -777.639
572.89 -781.64
567.443 -785.602
561.969 -789.527
556.468 -793.414
550.94 -797.263
545.386 -801.073
539.805 -804.844
534.198 -808.576
528.565 -812.27
522.907 -815.924
517.223 -819.539
511.514 -823.114
505.781 -826.649
500.023 -830.144
494.242 -833.6
488.436 -837.015
482.606 -840.389
476.754 -843.723
470.878 -847.017
464.98 -850.269
459.059 -853.48
453.116 -856.65
447.151 -859.779
441.164 -862.866
435.156 -865.911
429.127 -868.915
423.078 -871.876
417.007 -874.796
410.917 -877.673
404.807 -880.508
398.678 -883.3
392.529 -886.05
386.361 -888.757
380.174 -891.421
373.969 -894.041
367.746 -896.619
361.506 -899.153
355.248 -901.644
348.972 -904.092
342.68 -906.495
336.371 -908.855
330.046 -911.171
323.705 -913.443
317.349 -915.671
310.977 -917.854
304.59 -919.994
298.188 -922.088
291.772 -924.139
285.342 -926.144
278.898 -928.105
272.441 -930.021
265.97 -931.892
259.487 -933.718
252.991 -935.499
246.483 -937.235
239.962 -938.926
233.431 -940.571
226.888 -942.17
220.334 -943.725
213.769 -945.233
207.194 -946.696
200.61 -948.113
194.015 -949.485
187.411 -950.81
180.798 -952.09
174.176 -953.324
167.546 -954.511
160.908 -955.653
154.262 -956.748
147.608 -957.797
140.948 -958.8
134.28 -959.756
127.606 -960.666
120.926 -961.53
114.24 -962.347
107.549 -963.118
100.852 -963.842
94.151 -964.52
87.445 -965.151
80.735 -965.736
74.02 -966.273
67.303 -966.764
60.582 -967.209
53.858 -967.607
47.131 -967.958
40.402 -968.262
33.672 -968.519
26.939 -968.73
20.206 -968.894
13.471 -969.011
6.736 -969.081
-0.0 -969.104
-6.736 -969.081
-13.471 -969.011
-20.206 -968.894
-26.939 -968.73
-33.672 -968.519
-40.402 -968.262
-47.131 -967.958
-53.858 -967.607
-60.582 -967.209
-67.303 -966.764
-74.02 -966.273
-80.735 -965.736
-87.445 -965.151
-94.151 -964.52
-100.852 -963.842
-107.549 -963.118
-114.24 -962.347
-120.926 -961.53
-127.606 -960.666
-134.28 -959.756
-140.948 -958.8
-147.608 -957.797
-154.262 -956.748
-160.908 -955.653
-167.546 -954.511
-174.176 -953.324
-180.798 -952.09
-187.411 -950.81
-194.015 -949.485
-200.61 -948.113
-207.194 -946.696
-213.769 -945.233
-220.334 -943.725
-226.888 -942.17
-233.431 -940.571
-239.962 -938.926
-246.483 -937.235
-252.991 -935.499
-259.487 -933.718
-265.97 -931.892
-272.441 -930.021
-278.898 -928.105
-285.342 -926.144
-291.772 -924.139
-298.188 -922.088
-304.59 -919.994
-310.977 -917.854
-317.349 -915.671
-323.705 -913.443
-330.046 -911.171
-336.371 -908.855
-342.68 -906.495
-348.972 -904.092
-355.248 -901.644
-361.506 -899.153
-367.746 -896.619
-373.969 -894.041
-380.174 -891.421
-386.361 -888.757
-392.529 -886.05
-398.678 -883.3
-404.807 -880.508
-410.917 -877.673
-417.007 -874.796
-423.078 -871.876
-429.127 -868.915
-435.156 -865.911
-441.164 -862.866
-447.151 -859.779
-453.116 -856.65
-459.059 -853.48
-464.98 -850.269
-470.878 -847.017
-476.754 -843.723
-482.606 -840.389
-488.436 -837.015
-494.242 -833.6
-500.023 -830.144
-505.781 -826.649
-511.514 -823.114
-517.223 -819.539
-522.907 -815.924
-528.565 -812.27
-534.198 -808.576
-539.805 -804.844
-545.386 -801.073
-550.94 -797.263
-556.468 -793.414
-561.969 -789.527
-567.443 -785.602
-572.89 -781.64
-578.309 -777.639
-583.7 -773.601
-589.062 -769.525
-594.397 -765.412
-599.702 -761.262
-604.979 -757.076
-610.226 -752.853
-615.444 -748.593
-620.632 -744.298
-625.79 -739.966
-630.918 -735.599
-636.016 -731.196
-641.082 -726.758
-646.118 -722.284
-651.123 -717.776
-656.096 -713.233
-661.037 -708.656
-665.946 -704.044
-670.824 -699.399
-675.669 -694.719
-680.481 -690.006
-685.26 -685.26
-690.006 -680.481
-694.719 -675.669
-699.399 -670.824
-704.044 -665.946
-708.656 -661.037
-713.233 -656.096
-717.776 -651.123
-722.284 -646.118
-726.758 -641.082
-731.196 -636.016
-735.599 -630.918
-739.966 -625.79
-744.298 -620.632
-748.593 -615.444
-752.853 -610.226
-757.076 -604.979
-761.262 -599.702
-765.412 -594.397
-769.525 -589.062
-773.601 -583.7
-777.639 -578.309
-781.64 -572.89
-785.602 -567.443
-789.527 -561.969
-793.414 -556.468
-797.263 -550.94
-801.073 -545.386
-804.844 -539.805
-808.576 -534.198
-812.27 -528.565
-815.924 -522.907
-819.539 -517.223
-823.114 -511.514
-826.649 -505.781
-830.144 -500.023
-833.6 -494.242
-837.015 -488.436
-840.389 -482.606
-843.723 -476.754
-847.017 -470.878
-850.269 -464.98
-853.48 -459.059
-856.65 -453.116
-859.779 -447.151
-862.866 -441.164
-865.911 -435.156
-868.915 -429.127
-871.876 -423.078
-874.796 -417.007
-877.673 -410.917
-880.508 -404.807
-883.3 -398.678
-886.05 -392.529
-888.757 -386.361
-891.421 -380.174
-894.041 -373.969
-896.619 -367.746
-899.153 -361.506
-901.644 -355.248
-904.092 -348.972
-906.495 -342.68
-908.855 -336.371
-911.171 -330.046
-913.443 -323.705
-915.671 -317.349
-917.854 -310.977
-919.994 -304.59
-922.088 -298.188
-924.139 -291.772
-926.144 -285.342
-928.105 -278.898
-930.021 -272.441
-931.892 -265.97
-933.718 -259.487
-935.499 -252.991
-937.235 -246.483
-938.926 -239.962
-940.571 -233.431
-942.17 -226.888
-943.725 -220.334
-945.233 -213.769
-946.696 -207.194
-948.113 -200.61
-949.485 -194.015
-950.81 -187.411
-952.09 -180.798
-953.324 -174.176
-954.511 -167.546
-955.653 -160.908
-956.748 -154.262
-957.797 -147.608
-958.8 -140.948
-959.756 -134.28
-960.666 -127.606
-961.53 -120.926
-962.347 -114.24
-963.118 -107.549
-963.842 -100.852
-964.52 -94.151
-965.151 -87.445
-965.736 -80.735
-966.273 -74.02
-966.764 -67.303
-967.209 -60.582
-967.607 -53.858
-967.958 -47.131
-968.262 -40.402
-968.519 -33.672
-968.73 -26.939
-968.894 -20.206
-969.011 -13.471
-969.081 -6.736
-969.104 0.0
-969.081 6.736
-969.011 13.471
-968.894 20.206
-968.73 26.939
-968.519 33.672
-968.262 40.402
-967.958 47.131
-967.607 53.858
-967.209 60.582
-966.764 67.303
-966.273 74.02
-965.736 80.735
-965.151 87.445
-964.52 94.151
-963.842 100.852
-963.118 107.549
-962.347 114.24
-961.53 120.926
-960.666 127.606
-959.756 134.28
-958.8 140.948
-957.797 147.608
-956.748 154.262
-955.653 160.908
-954.511 167.546
-953.324 174.176
-952.09 180.798
-950.81 187.411
-949.485 194.015
-948.113 200.61
-946.696 207.194
-945.233 213.769
-943.725 220.334
-942.17 226.888
-940.571 233.431
-938.926 239.962
-937.235 246.483
-935.499 252.991
-933.718 259.487
-931.892 265.97
-930.021 272.441
-928.105 278.898
-926.144 285.342
-924.139 291.772
-922.088 298.188
-919.994 304.59
-917.854 310.977
-915.671 317.349
-913.443 323.705
-911.171 330.046
-908.855 336.371
-906.495 342.68
-904.092 348.972
-901.644 355.248
-899.153 361.506
-896.619 367.746
-894.041 373.969
-891.421 380.174
-888.757 386.361
-886.05 392.529
-883.3 398.678
-880.508 404.807
-877.673 410.917
-874.796 417.007
-871.876 423.078
-868.915 429.127
-865.911 435.156
-862.866 441.164
-859.779 447.151
-856.65 453.116
-853.48 459.059
-850.269 464.98
-847.017 470.878
-843.723 476.754
-840.389 482.606
-837.015 488.436
-833.6 494.242
-830.144 500.023
-826.649 505.781
-823.114 511.514
-819.539 517.223
-815.924 522.907
-812.27 528.565
-808.576 534.198
-804.844 539.805
-801.073 545.386
-797.263 550.94
-793.414 556.468
-789.527 561.969
-785.602 567.443
-781.64 572.89
-777.639 578.309
-773.601 583.7
-769.525 589.062
-765.412 594.397
-761.262 599.702
-757.076 604.979
-752.853 610.226
-748.593 615.444
-744.298 620.632
-739.966 625.79
-735.599 630.918
-731.196 636.016
-726.758 641.082
-722.284 646.118
-717.776 651.123
-713.233 656.096
-708.656 661.037
-704.044 665.946
-699.399 670.824
-694.719 675.669
-690.006 680.481
-685.26 685.26
-680.481 690.006
-675.669 694.719
-670.824 699.399
-665.946 704.044
-661.037 708.656
-656.096 713.233
-651.123 717.776
-646.118 722.284
-641.082 726.758
-636.016 731.196
-630.918 735.599
-625.79 739.966
-620.632 744.298
-615.444 748.593
-610.226 752.853
-604.979 757.076
-599.702 761.262
-594.397 765.412
-589.062 769.525
-583.7 773.601
-578.309 777.639
-572.89 781.64
-567.443 785.602
-561.969 789.527
-556.468 793.414
-550.94 797.263
-545.386 801.073
-539.805 804.844
-534.198 808.576
-528.565 812.27
-522.907 815.924
-517.223 819.539
-511.514 823.114
-505.781 826.649
-500.023 830.144
-494.242 833.6
-488.436 837.015
-482.606 840.389
-476.754 843.723
-470.878 847.017
-464.98 850.269
-459.059 853.48
-453.116 856.65
-447.151 859.779
-441.164 862.866
-435.156 865.911
-429.127 868.915
-423.078 871.876
-417.007 874.796
-410.917 877.673
-404.807 880.508
-398.678 883.3
-392.529 886.05
-386.361 888.757
-380.174 891.421
-373.969 894.041
-367.746 896.619
-361.506 899.153
-355.248 901.644
-348.972 904.092
-342.68 906.495
-336.371 908.855
-330.046 911.171
-323.705 913.443
-317.349 915.671
-310.977 917.854
-304.59 919.994
-298.188 922.088
-291.772 924.139
-285.342 926.144
-278.898 928.105
-272.441 930.021
-265.97 931.892
-259.487 933.718
-252.991 935.499
-246.483 937.235
-239.962 938.926
-233.431 940.571
-226.888 942.17
-220.334 943.725
-213.769 945.233
-207.194 946.696
-200.61 948.113
-194.015 949.485
-187.411 950.81
-180.798 952.09
-174.176 953.324
-167.546 954.511
-160.908 955.653
-154.262 956.748
-147.608 957.797
-140.948 958.8
-134.28 959.756
-127.606 960.666
-120.926 961.53
-114.24 962.347
-107.549 963.118
-100.852 963.842
-94.151 964.52
-87.445 965.151
-80.735 965.736
-74.02 966.273
-67.303 966.764
-60.582 967.209
-53.858 967.607
-47.131 967.958
-40.402 968.262
-33.672 968.519
-26.939 968.73
-20.206 968.894
-13.471 969.011
-6.736 969.081
0.0 969.104
6.736 969.081
13.471 969.011
20.206 968.894
26.939 968.73
33.672 968.519
40.402 968.262
47.131 967.958
53.858 967.607
60.582 967.209
67.303 966.764
74.02 966.273
80.735 965.736
87.445 965.151
94.151 964.52
100.852 963.842
107.549 963.118
114.24 962.347
120.926 961.53
127.606 960.666
134.28 959.756
140.948 958.8
147.608 957.797
154.262 956.748
160.908 955.653
167.546 954.511
174.176 953.324
180.798 952.09
187.411 950.81
194.015 949.485
200.61 948.113
207.194 946.696
213.769 945.233
220.334 943.725
226.888 942.17
233.431 940.571
239.962 938.926
246.483 937.235
252.991 935.499
259.487 933.718
265.97 931.892
272.441 930.021
278.898 928.105
285.342 926.144
291.772 924.139
298.188 922.088
304.59 919.994
310.977 917.854
317.349 915.671
323.705 913.443
330.046 911.171
336.371 908.855
342.68 906.495
348.972 904.092
355.248 901.644
361.506 899.153
367.746 896.619
373.969 894.041
380.174 891.421
386.361 888.757
392.529 886.05
398.678 883.3
404.807 880.508
410.917 877.673
417.007 874.796
423.078 871.876
429.127 868.915
435.156 865.911
441.164 862.866
447.151 859.779
453.116 856.65
459.059 853.48
464.98 850.269
470.878 847.017
476.754 843.723
482.606 840.389
488.436 837.015
494.242 833.6
500.023 830.144
505.781 826.649
511.514 823.114
517.223 819.539
522.907 815.924
528.565 812.27
534.198 808.576
539.805 804.844
545.386 801.073
550.94 797.263
556.468 793.414
561.969 789.527
567.443 785.602
572.89 781.64
578.309 777.639
583.7 773.601
589.062 769.525
594.397 765.412
599.702 761.262
604.979 757.076
610.226 752.853
615.444 748.593
620.632 744.298
625.79 739.966
630.918 735.599
636.016 731.196
641.082 726.758
646.118 722.284
651.123 717.776
656.096 713.233
661.037 708.656
665.946 704.044
670.824 699.399
675.669 694.719
680.481 690.006
685.26 685.26
690.006 680.481
694.719 675.669
699.399 670.824
704.044 665.946
708.656 661.037
713.233 656.096
717.776 651.123
722.284 646.118
726.758 641.082
731.196 636.016
735.599 630.918
739.966 625.79
744.298 620.632
748.593 615.444
752.853 610.226
757.076 604.979
761.262 599.702
765.412 594.397
769.525 589.062
773.601 583.7
777.639 578.309
781.64 572.89
785.602 567.443
789.527 561.969
793.414 556.468
797.263 550.94
801.073 545.386
804.844 539.805
808.576 534.198
812.27 528.565
815.924 522.907
819.539 517.223
823.114 511.514
826.649 505.781
830.144 500.023
833.6 494.242
837.015 488.436
840.389 482.606
843.723 476.754
847.017 470.878
850.269 464.98
853.48 459.059
856.65 453.116
859.779 447.151
862.866 441.164
865.911 435.156
868.915 429.127
871.876 423.078
874.796 417.007
877.673 410.917
880.508 404.807
883.3 398.678
886.05 392.529
888.757 386.361
891.421 380.174
894.041 373.969
896.619 367.746
899.153 361.506
901.644 355.248
904.092 348.972
906.495 342.68
908.855 336.371
911.171 330.046
913.443 323.705
915.671 317.349
917.854 310.977
919.994 304.59
922.088 298.188
924.139 291.772
926.144 285.342
928.105 278.898
930.021 272.441
931.892 265.97
933.718 259.487
935.499 252.991
937.235 246.483
938.926 239.962
940.571 233.431
942.17 226.888
943.725 220.334
945.233 213.769
946.696 207.194
948.113 200.61
949.485 194.015
950.81 187.411
952.09 180.798
953.324 174.176
954.511 167.546
955.653 160.908
956.748 154.262
957.797 147.608
958.8 140.948
959.756 134.28
960.666 127.606
961.53 120.926
962.347 114.24
963.118 107.549
963.842 100.852
964.52 94.151
965.151 87.445
965.736 80.735
966.273 74.02
966.764 67.303
967.209 60.582
967.607 53.858
967.958 47.131
968.262 40.402
968.519 33.672
968.73 26.939
968.894 20.206
969.011 13.471
969.081 6.736
969.104 0.0
969.081 -6.736
969.011 -13.471
968.894 -20.206
968.73 -26.939
968.519 -33.672
968.262 -40.402
967.958 -47.131
967.607 -53.858
967.209 -60.582
966.764 -67.303
966.273 -74.02
965.736 -80.735
965.151 -87.445
964.52 -94.151
963.842 -100.852
963.118 -107.549
962.347 -114.24
961.53 -120.926
960.666 -127.606
959.756 -134.28
958.8 -140.948
957.797 -147.608
956.748 -154.262
955.653 -160.908
954.511 -167.546
953.324 -174.176
952.09 -180.798
950.81 -187.411
949.485 -194.015
948.113 -200.61
946.696 -207.194
945.233 -213.769
943.725 -220.334
942.17 -226.888
940.571 -233.431
938.926 -239.962
937.235 -246.483
935.499 -252.991
933.718 -259.487
931.892 -265.97
930.021 -272.441
928.105 -278.898
926.144 -285.342
924.139 -291.772
922.088 -298.188
919.994 -304.59
917.854 -310.977
915.671 -317.349
913.443 -323.705
911.171 -330.046
908.855 -336.371
906.495 -342.68
904.092 -348.972
901.644 -355.248
899.153 -361.506
896.619 -367.746
894.041 -373.969
867.544 -474.427
