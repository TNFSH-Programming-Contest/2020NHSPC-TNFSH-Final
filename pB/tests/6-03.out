12041659.746